web_frontend/osmaxx/excerptexport/services/conversion_api_client.py

```diff
@@ -191,6 +192,12 @@ def update_order_status(self, extraction_order):
             if progress == 'successful':
                 self._download_result_files(extraction_order, job_status)
             elif self._extraction_processing_overdue(progress, extraction_order):
+                logger.warning(
+                    'Extraction order %s processing timeout overdue. Set status to FAILED. %s, %s',
+                    extraction_order.id,
+                    serializers.serialize('json', extraction_order),
```

Comment:
why would you want to log the entire extraction order in json? An ID and name does suffice, as we can always track this in the admin to see the details (using that ID).